src/codegen.cpp

```diff
@@ -6867,7 +6873,9 @@ static std::pair<std::unique_ptr<Module>, jl_llvm_functions_t>
                         }
                         new_union.TIndex = RTindex;
                     }
-                    V = new_union.Vboxed ? new_union.Vboxed : ConstantPointerNull::get(cast<PointerType>(T_prjlvalue));
+                    if (VN)
+                        V = (new_union.Vboxed ? new_union.Vboxed :
+                             ConstantPointerNull::get(cast<PointerType>(T_prjlvalue)));
```

Comment:
```suggestion
                        V = new_union.Vboxed ? new_union.Vboxed : V_rnull;
```

Reply:
Huh, I wonder how I originally missed this.... = = ....